modules/ROOT/pages/fd-scatter-gather.adoc

```diff
@@ -0,0 +1,259 @@
+
+.You can save data in one or more variables that one or more routes can use.
+image::fd-sg-alt-1.png[Saving data for use by more than one route]
+
+In this case, the first card in each route receives the same message, which might have these parts if you create two variables before the Scatter-Gather card:
+
+----
+payload
+attributes
+vars
+  var1
+  var2
+----
+
+You can use both variables in both routes, if you need to.
+
+The following configuration is useful if your variables need to be used only in single routes:
+
+.You can save data in a variable that is used by only one route.
+image::fd-sg-alt-2.png[Saving data for use by one route only]
+
+Suppose that you create `var1` at the start of the first route, and you create `var2` at the start of the second route. Then, the second card in the first route receives a message that has these parts:
+
+----
+payload
+attributes
+vars
+  var1
+----
+
+And the second card in the second route receives a message that has these parts:
+
+----
+payload
+attributes
+vars
+  var2
+----
+
+In both of the two previous diagrams, the final card could be a Transform card, which might require you to know how to write scripts in DataWeave, or another Set Variable card that, as in the example about aggregating contacts, combines somehow the results from the routes in the Scatter-Gather card and then passes the variable to another card, such as a Transform card.
+
+== Differences between Parallel For-Each, For-Each, and Scatter-Gather
+
+For-Each iterates through a collection of elements, processing one element at a time through a single route.
+
+Parallel For-Each iterates though a collection of elements _n_ times (where _n_ is less than or equal to the number of elements in the collection) with a single process. For example, if there are 10 elements and the maximum number of processes is set to 2, a Parallel For-Each card processes two elements at a time, five times.
+
+Scatter Gather runs two or more parallel processes, called _routes_. When all routes are finished, the results from each route are gathered together into a new payload.
+
+== Fields in a Scatter-Gather Card
+
+Timeout:: The maximum length of time to wait, in milliseconds, for the results from each of the parallel processes to be gathered together into a new payload and sent to the next card. A value of 0 or lower means that there is no timeout. The default value is 9,223,372,036,854,776,000 millisecond, which is the maximum value for a 64-bit float.
+
+Max concurrency:: The maximum number of routes to process in parallel. By default, all of the routes that you create in for an instance of the Scatter-Gather card run in parallel.
++
+The default value that appears in the card is 2,147,483,647. You do not need to worry about why this is the default. Just understand that, if you don't set the value to the number of routes in the scope of your Scatter-Gather card, the effective value is in fact the number of routes in the scope.
```

Comment:
Is this the default value in THIS example, or always?
Not sure what the focus of this para is, but if it can be reduced to the following, I recommend it:
```suggestion
Although the default value that appears in the card is 2,147,483,647, but if you don't set the value to the number of routes in the scope of your Scatter-Gather card, the effective value is the number of routes in the scope.
```
So...if you don't set a value, it uses the number of routes instead of the arbitrary default?